File: RefFiles/Telerik/UI for WinForms R3 2016/Examples/QuickStart/Tools/VisualStyleBuilder/LaunchForm.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace Telerik.Examples.WinControls.Tools.VisualStyleBuilder
    Partial Public Class LaunchForm
        Inherits Telerik.QuickStart.WinControls.CustomThemeExamplesLauncherForm
        Public Sub New()
            InitializeComponent()

            Dim resources As System.ComponentModel.ComponentResourceManager = New System.ComponentModel.ComponentResourceManager(GetType(LaunchForm))
            Me.pictureBoxLaunchExample.ButtonImage = CType(resources.GetObject("pictureBoxLaunchExample.Image"), System.Drawing.Image)
        End Sub
    End Class
End Namespace

File: RefFiles/Telerik/UI for WinForms R3 2016/Examples/QuickStart/Tools/ThemeViewer/LaunchForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace Telerik.Examples.WinControls.Tools.ThemeViewer
{
    public partial class LaunchForm : Telerik.QuickStart.WinControls.CustomThemeExamplesLauncherForm
    {
        public LaunchForm()
        {
            InitializeComponent();

            this.pictureBoxLaunchExample.ButtonImage = Telerik.Examples.WinControls.Properties.Resources.launch;
        }
    }
}
